programs/marinade-finance/src/events/admin.rs

```diff
@@ -54,13 +54,9 @@ pub struct InitializeEvent {
 #[event]
 pub struct EmergencyPauseEvent {
     pub state: Pubkey,
-    pub current_epoch: u64,
-    pub resume_at_epoch_change: U64ValueChange,
 }
 
 #[event]
 pub struct ResumeEvent {
     pub state: Pubkey,
-    pub current_epoch: u64,
```

Comment:
Will the current epoch be available/derivable from the event by default without including it like this? If not, I think it could be useful to have the epochs in the events.